web/source/kmwbase.ts

```diff
@@ -28,6 +28,20 @@
    Copyright 2017-2019 SIL International
 ***/
 namespace com.keyman {
+
+  export interface OptionType {
+    root: string;
+    resources: string;
+    keyboards: string;
+    fonts: string;
+    attachType: string; // 'auto', 'manual', 'default'
+    ui: string;
+    setActiveOnRegister: string; // TODO: Convert to boolean
```

Comment:
The option loader needs to be able to receive this as a string or a boolean when we do that change.

Reply:
I'll do that. Does @jahorton have any to add about `OptionType`?